.github/workflows/flutter-action.yml

```diff
@@ -0,0 +1,30 @@
+name: Flutter Action
+
+on:
+  push:
+    paths-ignore:
+    - 'documentation/**'
+    - 'LICENSE'
+    - 'README.md'
+
+jobs:
+  build:
+    runs-on: macOS-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: '12.x'
+    - uses: subosito/flutter-action@v1
+      with:
+        flutter-version: '1.22.5'
+        channel: 'stable'
+    - name: Run Flutter Commands
+      working-directory: ./app
+      run: |
+        flutter pub get
+        flutter format --set-exit-if-changed .
+        flutter analyze .
+        flutter test
+        flutter build apk
```

Comment:
These builds take a long time... maybe we should have a separate action for builds only when merging into `master`.